### DISCUSSION TITLE
GSoC: Reaching out to mentors of Project 13 ( A general OpenVINO performance benchmarking suite for Vision-LLM models)

### DISCUSSION THREAD

**Discussion by Felicia-fang** [openvinotoolkit/openvino#23215] [Google Summer of Code]
Hi, I'm Wu Fang. As a Master student at National university of Singapore specializing in Machine Intelligence and Applications, with a research focus on 3D vision detection tasks, I am keen on the "A general OpenVINO performance benchmarking suite for Vision-LLM models" project. My experience with Visual Question Answering and current work on 3D object detection pre-trained models positions me well to tackle this challenge. Could you specify if addressing certain issues or achieving specific benchmarks is part of the project’s scope? I'm keen to apply my insights to enhance the suite's capabilities.

**Comment by adrianboguszewski** [comment]
@junwenwu @helena-intel please look at it

**Reply by junwenwu** [reply]
@Felicia-fang Thank you for your interest. We are looking for a general performance benchmarking suite that can be easily used on OpenVINO vision-LLM models. This should be an enhancement to https://github.com/openvinotoolkit/openvino.genai with extended coverage of vision-LLM models as well as examples of variety of applications.
